src/test/java/uk/gov/companieshouse/itemgroupworkflowapi/integration/ItemGroupControllerIntegrationTest.java

```diff
@@ -0,0 +1,183 @@
+import uk.gov.companieshouse.itemgroupworkflowapi.model.ItemKind;
+import uk.gov.companieshouse.itemgroupworkflowapi.model.Links;
+import uk.gov.companieshouse.itemgroupworkflowapi.repository.ItemGroupsRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * Integration tests the {@link uk.gov.companieshouse.itemgroupworkflowapi.controller.ItemGroupController} class.
+ */
+@SpringBootTest
+@AutoConfigureMockMvc
+@ComponentScan("uk.gov.companieshouse.itemgroupworkflowapi")
+public class ItemGroupControllerIntegrationTest {
+
+    private static final String EXPECTED_ORDER_NUMBER = "123456";
+    private static final String VALID_DELIVERY_COMPANY_NAME = "Delivery Test Company";
+    private static final String VALID_ITEM_COMPANY_NAME = "Item Test Company";
+    private static final String VALID_COMPANY_NUMBER = "IG-12345-67890";
+    public static final String REQUEST_ID_HEADER_NAME = "X-Request-ID";
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper mapper;
+
+    @Autowired
+    private ItemGroupsRepository repository;
+
+    @AfterEach
+    void tearDown() {
+        repository.deleteAll();
+    }
+
+    @Test
+    @DisplayName("Create successful itemGroup - 201 Created")
+    void createItemGroupSuccessful_201Created() throws Exception {
+
+        // Given
+        final ItemGroupData newItemGroupData = createValidNewItemGroupData();
+
+        // When and Then
+        mockMvc.perform(post("/item-groups" )
+                        .header(REQUEST_ID_HEADER_NAME, "12345")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .accept(MediaType.APPLICATION_JSON)
+                        .content(mapper.writeValueAsString(newItemGroupData)))
+                .andExpect(status().isCreated())
+                .andDo(MockMvcResultHandlers.print());
+    }
+
+    @Test
+    @DisplayName("Create itemGroup unsuccessful - 400 Bad Request")
+    void createItemGroupUnsuccessful_400BadRequest() throws Exception {
+
+        // Given
+        final ItemGroupData newItemGroupData = createInvalidNewItemGroupData();
+
+        // When and Then
+        mockMvc.perform(post("/item-groups" )
+                        .header(REQUEST_ID_HEADER_NAME, "12345")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .accept(MediaType.APPLICATION_JSON)
+                        .content(mapper.writeValueAsString(newItemGroupData)))
+                .andExpect(status().isBadRequest())
+                .andDo(MockMvcResultHandlers.print());
+    }
+
+    @Test
+    @DisplayName("create duplicate itemGroup fails - 409 Conflict")
+    void createDuplicateItemGroupUnsuccessful_409Conflict() throws Exception {
```

Comment:
This test is currently failing and the test method name should be updated to remove the underscore to be `createDuplicateItemGroupUnsuccessful_409Conflict`